doc/setup.md

```diff
@@ -0,0 +1,13 @@
+# Setting up enclave-enabled EC2 host
+
+1. Go to `Launch an Instance` in the EC2 console
+2. Select `Amazon Linux 2 AMI (HVM) - Kernel 5.10, SSD Volume Type` under AMI
+3. Select `m5a.xlarge` under Instance Type. Complete list of instances which have Nitro Support can be found [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/instance-types.html#ec2-nitro-instances)
```

Comment:
I was going to suggest that `t4g.xlarge` is cheaper, but I just tried it and the console launch interface says it's not supported, [contrary to the documentation](https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave.html#nitro-enclave-reqs). Maybe depends on the region?

Anyway, this seems like a fine recommendation to start people.

Reply:
Yes, I have noticed the same issue with other instances, not sure if it is a region issue.